app/src/main/java/net/simplifiedcoding/multiviewlist/data/repository/Repository.kt

```diff
@@ -7,6 +7,7 @@ import javax.inject.Inject
 class Repository @Inject constructor(
     private val api: Api
 ) : SafeApiCall {
```

Comment:
What is this safeAPI call? 
 what it will do?
Why do we need it in our project ?